skyplane/cli/common.py

```diff
@@ -5,6 +5,7 @@
 
 import typer
 from rich.console import Console
+from rich import print as rprint
```

Comment:
Is there any reason you prefer `rprint` to `console.print` ? It will be good to use console.print for consistency.

Reply:
I use rprint inconsistently in several places throughout the code but also use console in some places. Ideally we should clean it up at some point.